commit 80b17ab573f2dec3e6635cc8d13d5690e66b9dfe
Merge: 217186d 625860f
Author: alexborro <alexborro@gmail.com>
Date:   Thu Jan 1 22:56:21 2015 -0200

    Merge pull request #1285 from drf5n/PID_CI_v2
    
    temperature.cpp: Fix typo of MAX_BED_PID to MAX_BED_POWER.

diff --git a/Marlin/temperature.cpp b/Marlin/temperature.cpp
index 5b116d9..ce9bba3 100644
--- a/Marlin/temperature.cpp
+++ b/Marlin/temperature.cpp
@@ -582,9 +582,9 @@ void manage_heater()
 		  temp_dState_bed = pid_input;
 
 		  pid_output = pTerm_bed + iTerm_bed - dTerm_bed;
-          	  if (pid_output > MAX_BED_PID) {
+          	  if (pid_output > MAX_BED_POWER) {
             	    if (pid_error_bed > 0 )  temp_iState_bed -= pid_error_bed; // conditional un-integration
-                    pid_output=PID_MAX;
+                    pid_output=PID_BED_POWER;
           	  } else if (pid_output < 0){
             	    if (pid_error_bed < 0 )  temp_iState_bed -= pid_error_bed; // conditional un-integration
                     pid_output=0;
